integration_tests/test-requirements-for-tox.txt

```diff
@@ -11,19 +13,19 @@ sh
 
 # for database tests
 psycopg2-binary
-sqlalchemy==1.2.18
-unidecode==1.0.23
-cheroot==6.5.4
+sqlalchemy
```

Comment:
I don't understand why the versions should be unfrozen. We should keep the code tested as close to production as possible, no?

Reply:
some cleanup to do to deduplicate